ensure_no_std/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "ensure_no_std"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+cairo-lang-utils = { path = "../crates/cairo-lang-utils", default-features = false }
+cairo-lang-casm = { path = "../crates/cairo-lang-casm", default-features = false }
+cairo-lang-vm-utils = { path = "../crates/cairo-lang-vm-utils", default-features = false, features = [
+    "execute_core_hints",
+] }
+cairo-lang-casm-contract-class = { path = "../crates/cairo-lang-casm-contract-class", default-features = false }
```

Comment:
You can add the  `features = ["serde"]` and any other feature that exists and should be supported in `no_std`